Formula/odin.rb

```diff
@@ -0,0 +1,37 @@
+class Odin < Formula
+  desc "The Odin Programming Language"
+  homepage "https://odin-lang.org/"
+  url "https://github.com/odin-lang/Odin/archive/v0.12.0.tar.gz"
+  sha256 "8356124c8cc7e08ac39872e5bb10593a412e67f81df621124097facd9b2b26cc"
+  head "https://github.com/odin-lang/Odin.git"
+
+  depends_on "llvm"
+
+  uses_from_macos "iconv"
```

Comment:
```suggestion
  uses_from_macos "libiconv"
```

We need this to fix warning:
```
==> FAILED
== /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/odin.rb ==
C: 10:  3: uses_from_macos should only be used for macOS dependencies, not iconv.
```

Reply:
Thanks for drawing my attention to it. I should've been more diligent!